test/Microsoft.Azure.SignalR.Tests/ClientInvocationManagerTests.cs

```diff
@@ -151,17 +160,18 @@ public async Task TestCompleteWithRouterServer(string protocol, bool isCompletio
         catch (Exception e)
         {
             Assert.False(isCompletionWithResult);
-            Assert.Equal(ErrorCompleteResult, e.Message);
+            Assert.Equal(CommonErrorCompleteResult, e.Message);
         }
     }
 
     [Fact]
     public void TestCallerManagerCancellation()
     {
-        var clientInvocationManager = GetTestClientInvocationManager();
+        var clientInvocationManager = GetTestClientInvocationManager(1);
         var invocationId = clientInvocationManager.Caller.GenerateInvocationId(TestConnectionIds[0]);
         var cts = new CancellationTokenSource();
         var task = clientInvocationManager.Caller.AddInvocation<string>("TestHub", TestConnectionIds[0], invocationId, cts.Token);
+        clientInvocationManager.Caller.SetAckNumber(invocationId, 1);
```

Comment:
all the test set acknumber manually? how we test if this SetAckNumber logic works well with invocation?